frontend/src/components/TaskCalendar.jsx

```
@@ -316,159 +891,42 @@ const TaskCalendar = () => {
-              className={styles.actionButton}
-              onClick={handleUpdateTask}
-            >
-              <Save size={16} />
-              <span>Save</span>
-            </button>
+            )
+          )}
+          <button
+            className={styles.actionButton}
+            onClick={isNewTask ? handleCreateTask : handleUpdateTask}
+          >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)